doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -100,6 +100,7 @@
 - Renamed `BaseRelativePointer` to `UntypedRelativePointer` [\#605](https://github.com/eclipse-iceoryx/iceoryx/issues/605)
 - Prevent building GoogleTest when `GTest_DIR` is defined [\#1758](https://github.com/eclipse-iceoryx/iceoryx/issues/1758)
 - Refactored `iceoryx_posh_testing` library into own CMakeLists.txt [\#1516](https://github.com/eclipse-iceoryx/iceoryx/issues/1516)
+- Change return type of `cxx::variant::emplace_at_index` and `emplace` to void [\#1394](https://github.com/eclipse-iceoryx/iceoryx/issues/1394)
```

Comment:
```suggestion
- Changed return type of `cxx::variant::emplace_at_index` and `emplace` to void [\#1394](https://github.com/eclipse-iceoryx/iceoryx/issues/1394)
```

Reply:
Throughout the file, the tense alternates between present and past so I would like to keep this line as it is.

Reply:
For me it makes more sense to use the past tense as the release note entries are typically about past changes. Happy to adapt the rest of the wording on a separate PR.

Reply:
I compared with older release notes - there we chose present tense consistently. I will adapt the whole file accordingly.